pkg/provision/primitives/volume.go

```diff
@@ -36,6 +43,9 @@ func (p *Provisioner) volumeProvisionImpl(ctx context.Context, reservation *prov
 		return VolumeResult{}, err
 	}
 
+	if config.Type != pkg.SSDDevice {
```

Comment:
Actually since now we only support one type, should we keep the `Type` field at all ?

Reply:
I was thinking the same, but then i thought may be we will need to revert that and support multiple types later again ?

Reply:
could be. And I guess it would make more sense from a compatibility point of view to not remove the field